### DISCUSSION TITLE
docs/cs/basics/tools

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#7] [CS]
# docs/cs/basics/tools

计算机专业入门的第一课通常被称之为“CS0”，课程名字为 “计算机科学导论” 或 “计算机科学概论”，这门课主要向没有学习过任何前置知识的新人介绍计算机科学是什么，当走进大学时应该学习什么，非常适合高考刚毕业的准大学生、大一新生、计算机自学爱好者等。

http://localhost:3000/docs/cs/basics/tools

**Comment by lidongyx** [comment]
测试
